Packs/FiltersAndTransformers/ReleaseNotes/1_2_71.md

```diff
@@ -0,0 +1,7 @@
+
+#### Scripts
+
+##### New: RemoveMatches
+
+- New: Removes items from the given value matching the provided list of patterns. If the matchexact Argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
```

Comment:
```suggestion
- New: Removes items from the given value matching the provided list of patterns. If the *matchexact* argument is 'yes', direct string compare is used, otherwise the comparison is done using regex.<~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>
```
Is this part of the text needed?  <~XSIAM> (Available from Cortex XSIAM %%XSIAM_VERSION%%).</~XSIAM>